paket.dependencies

```diff
@@ -1,4 +1,4 @@
-framework: auto-detect
+framework: netstandard2.0, net40, netcoreapp2.0, net461
```

Comment:
`auto-detect` no longer worked if I added `framework: netcoreapp2.0` to clitool dotnet-sourcelink. I added that so that the `paket.lock` file didn't pull in the tons of `netcoreapp1.0` dependencies. So here, we simply specify all the frameworks used in project, including test projects.